toDoApp/tsconfig.json

```diff
@@ -2,6 +2,7 @@
 {
   "compilerOptions": {
     /* Basic Options */
+    "include": ["./src/assets/images/images.d.ts"],
```

Comment:
mmm this shouldn't be here.
You need to extract the `images.d.ts` and put it inside a global folder called `types` and remove this line.